Tella/Scenes/Settings/Views/Servers/GDrive/CreateDriveFolder.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  CreateDriveFolder.swift
+//  Tella
+//
+//  Created by gus valbuena on 5/21/24.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import SwiftUI
+
+struct CreateDriveFolder: View {
+    @State var fieldContent : String = ""
+    var body: some View {
+        ContainerView {
+            VStack(spacing: 20) {
+                Spacer()
+                GDriveHeaderView(
+                    title: "Create new folder",
+                    subtitle: "Your reports will be uploaded to a new folder on your Google Drive. Choose a name for this folder here."
+                )
+                TextfieldView(fieldContent: $fieldContent, isValid: .constant(true), shouldShowError: .constant(false), fieldType: .text, placeholder: "Folder name")
```

Comment:
I think it's better to write the arguments on new lines to make them easier to read (Control - M)
  
                TextfieldView(fieldContent: $fieldContent,
                              isValid: .constant(true),
                              shouldShowError: .constant(false),
                              fieldType: .text,
                              placeholder: "Folder name")